SharedContent/cs-winui-template/WinUI3TemplateCs/App.xaml.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.UI.Xaml;
+
+
+namespace WinUI3TemplateCs
+{
+    public partial class App : Application
+    {
+        public App()
+        {
+            this.InitializeComponent();
+        }
+
+        protected override void OnLaunched(LaunchActivatedEventArgs args)
+        {
+            m_window = new MainWindow();
+            m_window.Activate();
+        }
+
+        private Window m_window;
```

Comment:
nit: fields should be on top of the class, above the constructor.  See [SA1201](https://documentation.help/StyleCop/SA1201.html).

Reply:
Also: we don't use hungarian prefixes in C#. You don't anywhere else in this project either. Can you name this `mainWindow`?